pkg/cnab/config-adapter/testdata/mybuns.bundle.json

```diff
@@ -9,6 +9,13 @@
       "image": "localhost:5000/mybuns:332dd75c541511a27fc332bdcd049d5b"
     }
   ],
+  "images": {
+    "whalesayd": {
+      "imageType": "docker",
+      "image": "carolynvs/whalesayd:latest",
+      "description": "Whalesay as a service"
```

Comment:
Shouldn't this have persisted the digest of the image now that your PR implementing tag resolution of referenced images during porter build was merged?

Reply:
I believe this testdata is used for testing the manifest to cnab bundle conversion which does not include the logic to generate the build.LOCAL_MANIFEST which is when the tag is resolved

Reply:
Gotcha, thanks! 👍